src/lib/components/workflow/workflow-relationships.svelte

```diff
@@ -52,7 +63,9 @@
         />
       {/if}
     </div>
-    {#if hasChildren}
+    {#if liveChildren.length}
+      <LiveChildWorkflowsTable children={liveChildren} />
```

Comment:
Do we want any kind of loading state on this table?

Reply:
I did it this way so that the "non"-live table shows up immediately (i.e. that's the loading state), and then the "live" table replaces it.